core/extension/gdextension.cpp

```diff
@@ -635,7 +635,13 @@ Ref<Resource> GDExtensionResourceLoader::load(const String &p_path, const String
 
 	Ref<GDExtension> lib;
 	lib.instantiate();
+#ifdef ANDROID_ENABLED
+	// On Android, the libraries are placed in the lib folder of the apk,
+	// so we only use relative paths here.
+	String abs_path = library_path.get_file();
```

Comment:
@konczg Looks like this was fixed already in https://github.com/godotengine/godot/pull/68362, so we should be able to revert this change.